<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAboutDialog</class>
 <widget class="QDialog" name="CAboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>261</width>
     <height>249</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>531</width>
     <height>3</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bug Report:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>280</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWeb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEMail">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>gzlldx@163.com</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
